settings.py

```diff
@@ -125,6 +125,7 @@
     'django.template.loaders.eggs.Loader',
 )
 
+# Add this to localsettings and set it to False, so that CSRF protection is enabled on localhost
 CSRF_ALWAYS_OFF = True
```

Comment:
@dimagi/team-commcare-hq We haven't turned on CSRF on prod. Everyone should add this to their `localsettings` and set it to `False` so that any new forms fail hard during dev. You can add csrf protection to forms by following these docs https://github.com/dimagi/commcare-hq/pull/9922/files#diff-94f3a4ae271e815b64fc9cf47ebb44d6R26